frontend/src/components/Library/types.ts

```diff
@@ -76,7 +75,16 @@ export interface IMessageOut {
   options?: IMessageOptions;
 }
 
-export type IResultType = ISQLQueryRunResult | ISQLQueryStringResult | ISelectedTablesResult | IChartGenerationResult;
+export enum StreamingEventType {
```

Comment:
Rename to something query specific - `QueryStreamingEvent` for ex (type is implicit)